src/content/developers/docs/consensus-mechanisms/index.md

```diff
@@ -40,49 +48,57 @@ A consensus mechanism in a cryptoeconomic system also helps prevent certain kind
 
-More on [proof-of-work (PoW)](/developers/docs/consensus-mechanisms/pow/)
+More on [proof-of-work](/developers/docs/consensus-mechanisms/pow/)
 
-### Proof of stake {#proof-of-stake}
+### Proof-of-stake {#proof-of-stake}
 
-Ethereum has plans to upgrade to a [proof-of-stake (PoS)](/developers/docs/consensus-mechanisms/pos/) consensus protocol.
+Ethereum has plans to upgrade to a **proof-of-stake (PoS)** consensus protocol.
 
 #### Block creation {#pos-block-creation}
 
-Proof of stake is done by validators who have staked ETH to participate in the system. A validator is chosen at random to create new blocks, share them with the network and earn rewards. Instead of needing to do intense computational work, you simply need to have staked your ETH in the network. This is what incentivises healthy network behaviour.
+Proof-of-stake is done by validators who have staked ETH to participate in the system. A validator is chosen at random to create new blocks, share them with the network and earn rewards. Instead of needing to do intense computational work, you simply need to have staked your ETH in the network. This is what incentivises healthy network behaviour.
 
 #### Security {#pos-security}
 
 A proof-of-stake system is kept secure by the fact that you'd need 51% of the total staked ETH to defraud the chain. And that your stake is slashed for malicious behaviour.
 
-More on [proof of stake](/developers/docs/consensus-mechanisms/pos/)
+More on [proof-of-stake](/developers/docs/consensus-mechanisms/pos/)
+
+### A visual guide {#types-of-consensus-video}
+
+Watch more on the different types of consensus mechanisms used on Ethereum:
+
+<iframe width="100%" height="315" src="https://www.youtube.com/embed/ojxfbN78WFQ" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
 
-### Sybil Resistance & Chain Selection
+### Sybil Resistance & Chain Selection {#sybil-resistance}
 
-Now technically, both proof-of-work (PoW) and proof-of-stake (PoS) are not consensus protocols by themselves, but they are often referred to as such for simplicity. PoW and PoS are actually Sybil resistance mechanisms and block author selectors, they are a way to decide who is the author of the latest block. It's this mechanism combined with at least a chain selection rule that makes up a true consensus mechanism.
+Now technically, proof-of-work and proof-of-stake are not consensus protocols by themselves, but they are often referred to as such for simplicity. They are actually Sybil resistance mechanisms and block author selectors; they are a way to decide who is the author of the latest block. It's this Sybil resistance mechanism combined with a chain selection rule that makes up a true consensus mechanism.
```

Comment:
```suggestion
Now technically, proof-of-work and proof-of-stake are not consensus protocols by themselves, but they are often referred to as such for simplicity. They are actually Sybil resistance mechanisms and block author selectors; they are a way to decide who is the author of the latest block. It's this sybil resistance mechanism combined with a chain selection rule that makes up a true consensus mechanism.
```
```suggestion
Now technically, proof-of-work and proof-of-stake are not consensus protocols by themselves, but they are often referred to as such for simplicity. They are actually Sybil resistance mechanisms and block author selectors; they are a way to decide who is the author of the latest block. It's this Sybil Resistance mechanism combined with a chain selection rule that makes up a true consensus mechanism.
```
I would say it should either be both capitalized for sybil resistance, or none in this case.